eth-reference-tests/src/referenceTest/java/tech/pegasys/artemis/reference/phase0/genesis/stateCheck.java

```diff
@@ -46,7 +46,7 @@
   @MethodSource({"genesisGenericInitializationSetup"})
   void genesisInitialization(BeaconState state) {
     final List<BLSKeyPair> validatorKeys =
-        new MockStartValidatorKeyPairFactory().generateKeyPairs(0, numValidators - 1);
+        new MockStartValidatorKeyPairFactory().generateKeyPairs(0, numValidators);
```

Comment:
I think this change generates numValidators + 1 number of validators, which is why we had a -1 previously. You sure you want to do that?